unblob/models.py

```diff
@@ -112,29 +152,14 @@ class Handler(abc.ABC):
     # (e.g. tar magic is in the middle of the header)
     YARA_MATCH_OFFSET: int = 0
 
+    EXTRACTOR: Extractor = NullExtractor()
```

Comment:
I think we should have some (hopefully integration?) test on a handler, which does not override the `EXTRACTOR` and uses `NullExtractor`, should not we?

Reply:
Frankly, I am inclined to remove this from this PR and introduce when needed.